cyclus_input/reactors/Arkansas_Nuclear_1.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation>
```

Comment:
hmmmmmm. I see.. interesting. it seems like you could not use the "/children/*" part of the include directive and then you wouldn't need these simulation tags.

(it just bugs me because the simulation tag will confuse people eventually. a single one of these reactor xml files will never be, on it's own, a valid simulation)